src/Illuminate/Support/Collection.php

```diff
@@ -920,6 +920,23 @@ public function slice($offset, $length = null)
         return new static(array_slice($this->items, $offset, $length, true));
     }
 
+     /**
```

Comment:
StyleCI aligned it to your method definition which is also indented incorrectly.